lib/accordion/readMe.md

```diff
@@ -0,0 +1,50 @@
+## Accordion
+
+A basic accordion component.
+
+### Properties
+| Name | Type | Default | Description |
+|:-----|:-----|:-----|:-----|
+| `data` | `Array` | &nbsp; | An array of contents for accordion. ( `Required` ) |
+| `headerKey` | `String` | `header` | Set a Key for the header of the accordion. | 
+| `contentKey` | `String` | `content` | Set a Key for the content of the accordion.|
+| `borderless` | `Boolean` | `false` | Prop to render borderless accordion |
+| `className` | `String` | &nbsp; | Set a class to style the Component|
+| `headerClassName` | `String` | &nbsp;| Set a class for the header.|
+| `contentClassName` | `String` | &nbsp;| Set a class for the content.|
+| `allowMultipleExpanded` | `Boolean` | `false`| allows to expend more the one item.|
```

Comment:
Typo: allows to `expand` more `than` one item.

Reply:
done! https://github.com/Codebrahma/react-lite-ui/pull/327/commits/88c9347111de6b454be7631c8350188fa2139310